config/jobs/kubernetes/sig-node/containerd.yaml

```diff
@@ -17,7 +17,7 @@ presets:
   - name: NON_MASQUERADE_CIDR
     value: 0.0.0.0/0
   - name: KUBELET_TEST_ARGS
-    value: --runtime-cgroups=/system.slice/containerd.service
+    value: "--runtime-cgroups=/system.slice/containerd.service --cgroup-driver=systemd"
```

Comment:
lgtm

Reply:
@akhilerm just out of curiosity, why do you quote value and why do you use double quotes? As far as I understand yaml doesn't require it. Is it some kind of test-infra specific convention?

Reply:
I saw that if multiple flags separated with spaces were given, some tests used `""`. I am also not sure if that is needed. So added it to be on the safer side. Didnt know yaml doesnt required the quotes. 

@bart0sh Should that be changed?

Reply:
@akhilerm It's up to you. It looks a bit strange, considering that yaml values are strings by default, but it's harmless.

Reply:
Done. Pushed the changes

Reply:
Can you take a look again on the changes ?

Reply:
/lgtm